modules/abstract-cosmos/src/cosmosCoin.ts

```diff
@@ -126,6 +126,10 @@ export class CosmosCoin extends BaseCoin {
     throw new Error('Method not implemented.');
   }
 
+  valuelessTransferAllowed(): boolean {
+    return true;
+  }
```

Comment:
@ajays97 it should be in osmo module.

Reply:
yes.

Reply:
Mark it false here, and true in osmo module.

Reply:
Makes sense. Adding it to only osmo for now.